lean4-input.el

```diff
@@ -253,15 +253,19 @@ tasks as well."
 
   (lean4-input-add-translations (mapcar (lambda (tr) (cons (car tr) (vconcat (cdr tr))))
                                         lean4-input-user-translations))
-  (lean4-input-add-translations (with-temp-buffer
-                                  (insert-file-contents (expand-file-name
-                                                         "abbreviations.json"
-                                                         lean4-input-data-directory))
-                                  (goto-char (point-min))
```

Comment:
You cannot remove this line because `json-parse-buffer` starts reading input at point.

Reply:
You mean this line?
```diff
-(goto-char (point-min))
```

It's actually OK. `insert-file-contents` in this case doesn't change the cursor. `json-read` also starts reading input at point. Have a look at https://github.com/emacs-mirror/emacs/blob/f32ce2e38cfc99d869a3ae07e912d7ce33772f12/lisp/json.el#L750

Reply:
> insert-file-contents in this case doesn't change the cursor.

I didn't know that. Thank you.